stdlib/platform/java/wyvern/String.wyv

```diff
@@ -11,5 +11,8 @@ def ofInt(x : Int) : String
 def ofFloat(f : Float) : String
   utils.ofFloat(f)
 
+def ofFormattedFloat(format : String, f : Float) : String
```

Comment:
Can you add a comment describing what the format string is?  OK to say it's the same as some function in the standard Java library.